enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -116,31 +145,40 @@ func (c *podmanCommandContainer) IsImageCached(ctx context.Context) (bool, error
 }
 
 func (c *podmanCommandContainer) PullImage(ctx context.Context, creds container.PullCredentials) error {
-	pullResult := runPodman(ctx, "pull", "", nil, nil, c.image)
+	pullResult := runPodman(ctx, "pull", nil, nil, c.image)
 	if pullResult.Error != nil {
 		return pullResult.Error
 	}
 	return nil
 }
-func (c *podmanCommandContainer) Start(ctx context.Context) error   { return nil }
-func (c *podmanCommandContainer) Remove(ctx context.Context) error  { return nil }
-func (c *podmanCommandContainer) Pause(ctx context.Context) error   { return nil }
-func (c *podmanCommandContainer) Unpause(ctx context.Context) error { return nil }
+
+func (c *podmanCommandContainer) Remove(ctx context.Context) error {
+	res := runPodman(ctx, "rm", nil, nil, "--force", c.name)
+	return res.Error
+}
+
+func (c *podmanCommandContainer) Pause(ctx context.Context) error {
+	res := runPodman(ctx, "pause", nil, nil, c.name)
+	return res.Error
+}
+
+func (c *podmanCommandContainer) Unpause(ctx context.Context) error {
+	res := runPodman(ctx, "unpause", nil, nil, c.name)
+	return res.Error
```

Comment:
nit: would be nice to micro-comment the nil, nils in here

runPodman(ctx, "unpause", nil /*=stdin*/, nil/*=stdout*/, c.name)

Reply:
Done